packages/smock/README.md

```diff
@@ -339,6 +308,20 @@ MyMockContract.smocked.myFunction.will.revert.with('0x1234')
 console.log(await MyMockContract.myFunction('Some return value!')) // Revert!
 ```
 
+### Sending transactions from a smocked contract
+```typescript
+import { ethers } from 'hardhat'
+import { smockit } from '@eth-optimism/smock'
+
+const myContractFactory = await ethers.getContractFactory('MyContract')
+const myContract = await myContractFactory.deploy(...)
+
+// Smockit!
+const mock = await smockit('MyContract')
```

Comment:
```suggestion
const mock = await smockit('AnotherContract')
```

The current example makes it seem like there's some relationship between mock and myContract, which I don't think is the case.